### PR TITLE
adds narsie plushie and ratvar plushie on chaplain vendor

### PR DESCRIPTION
**Pull request by LetterN**
:cl:
add: Adds ratvar plushie and narsie plushie on chaplain vendors
/:cl:

[why]: why isn't this a thing yet?

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
too op

**Comment by Iamgoofball** [comment]
this should probably be contraband, not premium

**Comment by 81Denton** [comment]
Don‘t those two cause an explosion when they‘re next to each other? Or is it a subtype

**Comment by LetterN** [comment]
it makes a small bomb. and you can basicaly use it only one time, because it deletes the one loose

**Comment by Anonmare** [comment]
I still want de-powered oldcult clothes (the red robes and bronze stuff) for the Chaplain

**Comment by Anonmare** [comment]
Yeah but it's not got the same level as armour as the Chappy's usual loadout

File: code/modules/vending/wardrobes.dm [hunk 1]
```diff
@@ -294,7 +294,9 @@
 					/obj/item/clothing/suit/nun = 1,
 					/obj/item/clothing/head/nun_hood = 1,
 					/obj/item/clothing/suit/holidaypriest = 1,
-					/obj/item/storage/fancy/candle_box = 2)
+					/obj/item/storage/fancy/candle_box = 2)			
+	premium = list(/obj/item/toy/plush/plushvar = 1,
```
Comment (ninjanomnom) [inline code/modules/vending/wardrobes.dm]: contraband

**Review by ninjanomnom — DISMISSED** [review]

**Comment by LetterN** [comment]
@ninjanomnom done

**Review by Cyberboss — DISMISSED** [review]

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**